src/haystack/analyzer.clj

```diff
@@ -120,17 +120,27 @@
     (flag-frame frame :repl)
     frame))
 
+(defn- tool? [frame-name last?]
```

Comment:
Why not `tooling-frame?`?

Reply:
Also - it might be good to explain in the docstring the current criteria for a tooling frame.